Corona-Warn-App/src/main/res/layout/fragment_submission_done_no_consent.xml

```diff
@@ -0,0 +1,68 @@
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true"
+        tools:context=".ui.submission.fragment.SubmissionDoneNoConsentFragment">
+
+        <include
+            android:id="@+id/submission_done_no_consent_header"
+            layout="@layout/include_header"
+            android:layout_width="@dimen/match_constraint"
+            android:layout_height="wrap_content"
+            app:icon="@{@drawable/ic_close}"
+            app:title="@{@string/submission_done_no_consent_title}"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <include
+            android:id="@+id/submission_done_no_consent_content"
+            layout="@layout/include_submission_done_no_consent"
+            android:layout_width="@dimen/match_constraint"
+            android:layout_height="@dimen/match_constraint"
+            app:layout_constraintBottom_toBottomOf="@id/guideline_action"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/submission_done_no_consent_header" />
+
+        <Button
+            android:id="@+id/submission_done_button_continue_with_symptom_recording"
+            style="@style/buttonPrimary"
+            android:layout_width="@dimen/match_constraint"
+            android:layout_height="wrap_content"
+            android:text="@string/submission_done_no_consent_continue_with_symptom_recording"
+            app:layout_constraintBottom_toTopOf="@+id/submission_contact_button_enter"
+            app:layout_constraintEnd_toStartOf="@+id/guideline_end"
+            app:layout_constraintStart_toStartOf="@id/guideline_start"
+            app:layout_constraintTop_toBottomOf="@+id/submission_done_no_consent_content"/>
+        <Button
+            android:id="@+id/submission_done_contact_button_break"
+            style="@style/buttonPrimary"
+            android:layout_width="@dimen/match_constraint"
+            android:layout_height="wrap_content"
+            android:text="@string/submission_done_no_consent_break_flow"
+            android:textAllCaps="true"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@+id/guideline_end"
+            app:layout_constraintStart_toStartOf="@id/guideline_start"
+            app:layout_constraintTop_toBottomOf="@+id/submission_done_button_continue_with_symptom_recording" />
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guideline_action"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintGuide_end="119dp" />
```

Comment:
119dp is odd. Please use even numbers, ideally a multiple of 8, so 120 would be good.

Reply:
noted